### ISSUE TITLE
[Docker] Incorrect latest tag for tendermint/tendermint

### ISSUE THREAD

**Issue by jstuczyn** [tendermint/tendermint#3671]
Hi!

For a while I've been trying to figure out, why calling `tendermint init` using the `latest` version of tendermint/tendermint docker container was producing different set of `consensus_params`:

```
"consensus_params": {
    "block_size": {
      "max_bytes": "22020096",
      "max_gas": "-1"
    },
    "evidence": {
      "max_age": "100000"
    },
    "validator": {
      "pub_key_types": [
        "ed25519"
      ]
    }
  },
```

as opposed to the one run locally (0.31.5-4253e67c):
```  
"consensus_params": {
    "block": {
      "max_bytes": "22020096",
      "max_gas": "-1",
      "time_iota_ms": "1000"
    },
    "evidence": {
      "max_age": "100000"
    },
    "validator": {
      "pub_key_types": [
        "ed25519"
      ]
    }
  },
```

The different name of `block` vs `block_size` was preventing a node from startup, what took me a while to notice...

Interestingly testnet command completely ignores `consensus_params` from the genesis file and all runs just fine.

Now, it turns out the `latest` tag refers to build with version `0.30.4-bf23f747`. Calling docker with the explicit tag, i.e. `tendermint/tendermint:v0.31.5` fixes the mentioned issue.
So basically `latest` needs to refer to actually the most recent build.

**Comment by melekes** [comment]
> So basically latest needs to refer to actually the most recent build.

@greg-szabo should we link latest to newest tag when we deploy?

```
docker tag tendermint:vX.Y.Z tendermint:latest
```

**Comment by greg-szabo** [comment]
Hi jstuczyn,

Thanks for opening the issue. The v0.30.4 was released around the time we put a new release management process in place. The new process uses the latest tag for the last build, however, I can't recall if v0.30.4 was created with that or if we maybe did something manually and missed this bit.
Apologies for the lost time and I hope it didn't derail you from exploring tendermint.

Also, we regularly release new versions of tendermint. If you want to make sure you are always on the latest version, do regular `docker pull tendermint/tendermint`s because docker doesn't update downloaded images automatically when you run `docker run`. (It only does an implicit pull when the image doesn't exist locally.)

I believe the issue is fixed, but if you still experience it, feel free to reopen the ticket. If you experience any other issues, please open a new ticket.

Regards,
Greg

PS:
Release process build step here: https://github.com/tendermint/tendermint/blob/master/.circleci/config.yml#L356

Current latest will report version 0.31.7.
